yb-voyager/cmd/common.go

```diff
@@ -225,14 +234,21 @@ func displayExportedRowCountSnapshot(snapshotViaDebezium bool) {
 					schema = tableParts[0]
 					table = tableParts[1]
 				}
-				uitable.AddRow(schema, table, exportedRowCount[key])
+				displayTableName := table
```

Comment:
I see this logic being used in multiple places. Can we define a func getDisplayTableName(table, sourceDBType, leafPartitions, ... ) ?

Reply:
Yes will do that, this is the code duplication mainly I was talking about.